### ISSUE TITLE
MiWoEx Onthologie

### ISSUE THREAD

**Issue by eriktamsen** [BAMresearch/LebeDigital#87]
This issue is related to the added concrete onthologies. We discussed this in the last Knowledge Graph meeting.
I am creating this to organize our branches, issues, etc.

This is the comment in the related PR from Jörg. 

 T[here](https://github.com/BAMresearch/LebeDigital/blob/19405e933a5b26d5aafb8237e12f9c6701338960/lebedigital/mapping/emodul_mapping.py#L116) are a couple of problems. 
1) One is related to the fact that in the mapping procedure some of the deleted ontologies are still imported e.g. MergedAllCoreOntology.owl , EM.xml, ...
2) There is also the mseo_mid that is loaded, but owlready requires this in owl format (not in ttl as given here). 

Could you please discuss with @AidaZt or (@soudehMasoudian @alFrie) to fix this. Currently, the tests are not working and thus I can't merge into the main branch.

**Comment by eriktamsen** [comment]
@mattheokru: we discussed the files in the ConcreteOntologie directory.
Two comments: 1) please move the data file, even if it is just for testing purposes to a relevant usecase, or even a new testing usecase.
2) it would be nice to rename the files to something understandable. So instead of CPTO, rather ConcreteProcess........, instead of EM or youngs_modulus_ontologie or something.

**Comment by joergfunger** [comment]
1. Turtle is fine with me when working with the mapping. Not sure though why we would have to reopen it in protege. For me, the way to work with that would be to create all new classes and properties in protege (using owl). Once this is done, we would use the diagrams.net approach with the ontodocker to generate the T-Box and use the export with placeholders to generate the file (ttl) which we could use for the mapping (which then is just replace placeholder $author by the name that we have in the yaml file.
2. I don't see how we could put these all together. I think we agreed to have first the mixture being mapped, then the compression tests, and then the Young's modulus test. They are referring to each other (all are related to the mix), that's why the identifier to link them together is important but they should not be mapped together. Also in the future, people would generate the mix and upload it, and only afterwards there will be the compression test (also uploaded) and based on that result they would know what is the load to be applied to the Youngs modulus test. As such, at the end there is one KG. But each mapping introduces a subgraph  is generated from the A-Box mapping, which then have to be joined/uploaded to the global triple store.
3. yes
4. From my point of view no, I would suggest focusing only on the new way. But please discuss that also with @AidaZt and Thilo.
Happy holidays and happy new year :)

**Comment by joergfunger** [comment]
All the different tests should finally be in the same graph (otherwise you can't query them jointly). But in order to get there, I think we would have to decompose the problem into first creating the sub graph related to one step of an experiment (so e.g. mix, compression, elasticity) and then upload that to the joint KG.
